### ISSUE TITLE
Used term "native" framework is consistent.

### ISSUE THREAD

**Issue by JackGrinningCat** [NuGet/docs.microsoft.com-nuget#190]
Hey, 

I want to control my dependencies in an interop (C++/CLI) package. Like described here
https://docs.microsoft.com/en-us/nuget/schema/nuspec#dependencies

https://docs.microsoft.com/en-us/nuget/create-packages/native-packages
On this page the the term, "native Framework" is used, but if you check the article and the use of group those dependecies to used framework the listing (https://docs.microsoft.com/en-us/nuget/schema/target-frameworks) is used.

<dependencies>
   <group targetFramework="native">
      <dependency id="boost" version="1.62.0.0" />
   </group>
</dependencies> 

That again not mentioning a "native" framework. Sure i could try on my own and will sure enough, but I miss some clarity in the use of common term. 
Basically i wish it would working even if not documented that it should :)

If native is a not supported framework, then please don't call it a "native" framework, or if it is a supported framework, take it into the targetFramework list. If it is only supported in another sense, clarify the use of term framework.

**Comment by JackGrinningCat** [comment]
I had an example 

`<dependencies>
   <group targetFramework="native">
      <dependency id="boost" version="1.63.0"/>
   </group>
</dependencies>`

**Comment by karann-msft** [comment]
@kraigb can you please take a look.

**Comment by CamSoper** [comment]
Rolling up to #238
